samples/ActionScript/Functional.as

```diff
@@ -0,0 +1,5 @@
+// AS3: Unique (from Angelscript) function declaration syntax using the function keyword
+
+package{
+	function foo();
+}
```

Comment:
Could you remove those files as well? If you think they are needed for tests because they match some of the new patterns the larger files don't match, you can move them to `test/fixtures/`.